spring-cloud-stream-app-starters-docs/pom.xml

```diff
@@ -196,7 +196,7 @@
 								</goals>
 								<configuration>
 									<target>
-										<zip destfile="${project.build.directory}/spring-cloud-stream-modules-${project.version}.zip">
+										<zip destfile="${project.build.directory}/spring-cloud-stream-app-starters-docs-${project.version}.jar">
```

Comment:
I think the prefix `spring-cloud-stream-modules` takes into account what artifactId would normally provide.  The version seems correct to use here imho